modules/data-model/links/api/src/main/java/io/uhndata/cards/links/api/LinkUtils.java

```diff
@@ -56,6 +59,25 @@ public interface LinkUtils
      */
     Collection<Link> getLinks(Node source);
 
+    /**
+     * Retrieve all the links from a resource of a desired link type.
+     *
+     * @param source the node to get links from
+     * @param linkName The name of the link type, or a path starting from the root node to the link typs
+     * @return a collection of links, may be empty
+     */
+    Collection<Link> getLinksOfType(Node source, String linkName);
```

Comment:
```suggestion
    Collection<Link> getLinksOfType(Node source, String type);
```

And rename all the other places where this param is used)